examples/sample-with-dns-zone.tfvars

```diff
@@ -0,0 +1,237 @@
+#-----------------------------------------------------------------------------
+
+# Name of the A record, of following dns zone, where the ecl watch ip is placed
+# This A record will be created and therefore should not exist in the following
+# dns zone.
+
+a_record_name="hpcc"
```

Comment:
Make this an empty string value and include an example in the preceding comments, like the existing values; e.g.

\# Example entry: "hpcc"

However, should we perhaps also note in the directions that this should be something project-specific rather than generic?  Like 'play' would be used for our play cluster.